index.html

```diff
@@ -0,0 +1,130 @@
+    <title>Form Project</title>
+    <link href="CSS/style.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b77bd6f2e2.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="container">
+        <header class="heading">
+            <div class="sign-in-details">
+                <p>Already have an Intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a>
+            </div>
+            <div class="sign-in">
+                <button type="button"><i class="fas fa-lock"></i>Sign In</button>
+            </div>
+            
+        </header>
+
+        <main class="main-one">
+            <div class="form-heading">
+                <div class="form-h1">
+                    <img src="images/Heading.PNG" alt="Heading">
+                </div>
+                
+                <div class="f-images">
+                    <div class="form-images">
+                        <div class="header-icon">
+                            <img src="images/header-icon-01.svg" alt="Sub-Header-01">
+                        </div>
+                        <div >
+                            <img src="images/text-01.svg" alt="Pic-01">
+                        </div>
+                        
+                    </div>
+                    
+                    <div class="form-images">
+                        <div class="header-icon"> 
+                            <img src="images/header-icon-02.svg" alt="Sub-Header-02">
+                        </div>
+                        <div>
+                            <img src="images/text-02.svg" alt="Pic-02"> 
+                        </div>
+                            
+    
+                    </div>
+                    
+                    <div class="form-images">
+                        <div class="header-icon"> 
+                            <img src="images/header-icon-03.svg" alt="Sub-Header-03">
+                        </div>
+                        <div>
+                            <img src="images/text-03.svg" alt="Pic-03"> 
+                        </div>
+                    </div>
+    
+                </div>
+                
+            </div>
+
+            <form action="#" method="POST">
+                <div class="div-one">
+                    <h2>Create an Intuit account</h2>
+                    <p>One account for everything Intuit, including Mint.</p>
+                    <a href="#">Learn more</a>
+                </div>
+                
+                <div class="data-input">
+                    <div class="e-mail">
+                        <label for="e-mail">Email address</label>
+                        <input id="e-mail" type="email" name="e-mail">
+                    </div>
+                    
+                    <div class="phone">
+                            <label for="phone">Phone (recommended)</label>
+                            <input id="phone" type="text" name="phone">
```

Comment:
- [ ] The type of the phone `<input>` should be `tel`, not text ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/)) 📞